lib/valueflow.cpp

```diff
@@ -1208,7 +1208,7 @@ static void valueFlowImpossibleValues(TokenList& tokenList, const Settings& sett
     for (Token* tok = tokenList.front(); tok; tok = tok->next()) {
         if (tok->hasKnownIntValue())
             continue;
-        if (Token::Match(tok, "true|false"))
+        if (tok->tokType() == Token::Type::eBoolean)
```

Comment:
Should use `%bool%`. There's also another instance which matches the literals in this file.